cypress/support/pageObjects/inventoryPage.js

```diff
@@ -0,0 +1,88 @@
+var inventoryItem = require('../../models/inventoryItem.json');
+
+export class InventoryPage {
+    static wrapItems() {
```

Comment:
Such a long method. Consider splitting it to smaller ones

Reply:
Method was split into three (Item, items and locators)

Reply:
Method was split into three (Item, items and locators)